### DISCUSSION TITLE
The theme selector spams the search results

### DISCUSSION THREAD

**Discussion by dvarrazzo** [pradyunsg/furo#240] [Potential Issue] [answered]
This page is generated with furo 2021.8.31:

https://www.psycopg.org/psycopg3/docs/search.html?q=get_key&check_keywords=yes&area=default

The summary for the search hits are "Toggle Light / Dark / Auto color theme Toggle table of contents sidebar..."

![image](https://user-images.githubusercontent.com/199429/132362919-e7898557-f84b-42ce-83f5-d37651794e2d.png)

**Comment by pradyunsg** [comment]
Ahahahahahaahah, nice catch!

Sphinx’s search is finding screen reader only content. This one is gonna be a doozy to figure out.

**Comment by pradyunsg** [comment, marked as answer]
This should be fixed in the "Balmy Blue" release (https://pradyunsg.me/furo/changelog/#balmy-blue). :)

**Reply by dvarrazzo** [reply]
Thank you very much: [it works as expected](https://www.psycopg.org/psycopg3/docs/search.html?q=get_key&check_keywords=yes&area=default) now :)
